variables.tf

```diff
@@ -50,3 +39,34 @@ variable "automation_account" {
   description = "Automation account where the resource graph script will be deployed."
   nullable    = false
 }
+
+variable "event_pipeline_config" {
+  type = object({
+    enabled     = bool
+    name        = optional(string, "QBY EventPipeline")
+    service_uri = optional(string)
+  })
+
+  description = <<-DOC
+  ```
+  {
+    enabled       = Enable the action group if you want to send data to a monitoring service.
+    name          = Name of the alert webhook.
+    service_uri   = Link to the webhook receiver URL. Must contain the placeholder \"{{secret}}\". This placeholder will be replaced by the secret value from var.secret. This is used to add authentication to the webhook URL as a query parameter.
```

Comment:
What happens if it doesn't contain `{{secret}}`?

Reply:
The validation in line 68 fails. It is assumed that the webhook is secured with a secret so the URI must contain the placeholder.